cli/cmd/host/update.go

```diff
@@ -0,0 +1,32 @@
+package host
+
+import (
+	"encoding/json"
+	"log"
+	"os"
+
+	"github.com/gravitl/netmaker/cli/functions"
+	"github.com/spf13/cobra"
+)
+
+var hostUpdateCmd = &cobra.Command{
+	Use:   "update HostID /path/to/host_definition.json",
```

Comment:
how do i know the fields of the json file?

Reply:
will add a sample definition json once I get it